src/content/en/2020/capabilities.md

```diff
@@ -0,0 +1,336 @@
+- [Web NFC](https://web.dev/nfc/), [Web Serial](https://web.dev/serial/), Web USB, Web Bluetooth, and other APIs (for the entire list, see the [Fugu API Tracker](https://goo.gle/fugu-api-tracker))
+
+Everyone can propose a new capability by [creating a ticket in the Chromium bug tracker](https://bit.ly/new-fugu-request). The Chromium contributors examine the proposals and discuss all APIs with other developers and browser vendors through the appropriate standards bodies. Meanwhile, the Fugu team implements the API in Chromium, where it is initially implemented behind a flag. In the further process, the API is made available to a limited audience via an [origin trial](https://web.dev/origin-trials/). During this phase, developers can sign up for a token to test the API on a specific origin. If the API turns out to be robust enough, the API ships in Chromium and, if the vendors decide so, other browsers. The [Capability Status](https://web.dev/fugu-status/) site shows where the different Capability APIs are in the process.
+
+The codename of the Capabilities project is named after a Japanese dish: Correctly prepared, the meat of the blowfish is a special taste experience. If prepared incorrectly, however, it can be fatal. The powerful APIs of Project Fugu are extremely exciting for developers. However, they can affect the security and privacy of the user. Therefore, the Fugu team pays special attention to these issues: For instance, new interfaces require the website to be sent over a secure connection (HTTPS). Some of them require a user gesture, such as a click or key press, to prevent fraud. Other capabilities require explicit permission by the user. Developers can use all APIs as a progressive enhancement: By feature detecting the APIs, applications won't break in browsers lacking support for those capabilities. In browsers that support them, users can get a better user experience. This way, web apps [progressively enhance](https://web.dev/progressively-enhance-your-pwa/) according to the user's particular browser.
+
+This chapter gives an overview of various modern web APIs, and the state of web capabilities in 2020 based on usage data by the HTTP Archive. Due to technical limitations, the HTTP Archive only has data available for APIs that require neither permission, nor a user gesture. Where no data is available, the percentage of page loads in Google Chrome according to [Chrome Platform Status](https://chromestatus.com/metrics/feature/timeline/popularity) will be shown instead. Since some interfaces are brand-new and their usage is still very low, the statistics are not necessarily meaningful. Unlike most chapters, stats will be presented as the absolute numbers of pages rather than relative percentages, where available. However, in many cases trends can still be read from the data. Unless otherwise noted, the APIs are only available in Chromium-based browsers, and their specifications are in the early stages of standardization.
```

Comment:
One other point that could be good to make here is that measuring low adoption just means that we have a baseline from which to compare future growth. Future annual editions of this chapter can look back to these stats to see how much the APIs have matured and improved their adoption.